tests/mountingOptions/stubs.global.spec.ts

```diff
@@ -300,4 +301,47 @@ describe('mounting options: stubs', () => {
 
     expect(wrapper.html()).toBe('<foo-bar-stub></foo-bar-stub>')
   })
+
+  describe('stub slots', () => {
+    const Component = {
+      name: 'Parent',
+      template: `
+        <div>
+          <component-with-slots>
+            <template #default>Default</template>
+            <template #named>A named</template>
+            <template #noop>A not existing one</template>
+            <template #scoped="{ boolean, string }">{{ boolean }} {{ string }}</template>
+          </component-with-slots>
+        </div>`,
+      components: { ComponentWithSlots }
+    }
+
+    afterEach(() => {
+      config.renderAllStubSlots = false
+    })
+
+    it('renders only the default stub slot by default', () => {
+      const wrapper = mount(Component, {
+        global: {
+          stubs: ['ComponentWithSlots']
+        }
+      })
+      expect(wrapper.html()).toBe(
+        `<div><component-with-slots-stub>Default</component-with-slots-stub></div>`
```

Comment:
This seems in correct - passing a stub when the `renderAllStubSlots` is `false` (whcih is it in this test) should just render a stub with no children, shouldn't it? (not sure how beta works here, but if this is how it worked, it's a bug in beta).

Reply:
I thought that it made sense to render the default slot always. If you have a button or something of the like, most often the default slot is what you need.

Reply:
I think this behavior be part of the `renderAllStubSlots` flag - it seems much cleaner to me to just do

- default behavior: a stub is a stub. It just stubs everything out. easy to understand
- `renderAllStubSlots`: renders the slots (default and named). also easy to understand.

We could also make the flag `renderStubSlots` (this way, `all` really does mean all slots, not just named slots).